content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -231,6 +237,46 @@ livenessProbe:
     port: liveness-port
 ```
 
+## Protect slow starting containers with startup probes
+
+Sometimes, you have to deal with legacy applications that might require
+an additional startup time on their first initialization.
+In such cases, it can be tricky to setup liveness probes parameters without
```

Comment:
```suggestion
 In such cases, it can be tricky to setup liveness probe parameters without
```
maybe? Not sure what the right wording is here.

Reply:
I agree, strangely enough they tend to put probes everywhere, even when it doesn't make sense... Let's put some correct English :)